tools/crates-validator/src/lib.rs

```diff
@@ -466,87 +482,108 @@ impl Validator {
         info!("running: {}", cmd.display());
         let mut cmd = Command::new(cmd);
         let mut cmd = cmd.current_dir(working_dir);
-        if self.context.fast_mode {
+        if profile.settings.fast_mode {
             cmd = cmd.env("RISC0_EXPERIMENTAL_PREFLIGHT", "1")
```

Comment:
This flag no longer exists, we should replace it with RISC0_DEV_MODE=1 for execute only mode.

Reply:
Thanks for pointing out!
Will be addressed in #957